packages/p2p/src/stores/general-store.js

```diff
@@ -433,6 +433,7 @@ export default class GeneralStore extends BaseStore {
     }
 
     onMount() {
+        this.setIsLoading(true);
```

Comment:
where do we set this to `false`?

Reply:
@farrah-deriv it's on master branch
mmm
i removed it in previous commits, but reverted my change in the last one
there is no changes in PR for this line of code

Reply:
ok, i have updated the branch and it becomes outdated